services/frontend/src/components/bp-chip-avatar/index.js

```diff
@@ -43,4 +41,4 @@ ProducerChipAvatar.propTypes = {
   defaultName: PropTypes.string
 }
 
-export default ProducerChipAvatar
+export default React.memo(ProducerChipAvatar)
```

Comment:
What if we import memo like `import React, { memo } from 'react'`

Reply:
Yes, you right!